### PR TITLE
Automate bumping extension version across multiple files

### PR DESCRIPTION
**Pull request by JamesGuthrie**
## Description

As in title.

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] ~~CHANGELOG entry for user-facing changes~~
- [x] Updated the relevant documentation

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -4,19 +4,16 @@ The following are step-by-step instructions for releasing the Promscale extensio
...
 - [ ] Ensure `upgradeable_from` in `templates/promscale.control` contains the previously released version.
-- [ ] Update `CHANGELOG.md` with release version and date.
-- [ ] Update the package version in `Cargo.toml` to the version being released
-- [ ] Update the version in the references to the .so in `migration/incremental/001-extension.sql`
-- [ ] Update the MD5 hash of the `migration/incremental/001-extension.sql` file in `e2e/tests/incremental-freeze-test.rs`
-- [ ] Update the version in the references to the .so in `migration/idempotent/014-extension-type-functions.sql`
-- [ ] Update the version in the references to the .so in `sql/promscale--0.0.0.sql`
+- [ ] Update `CHANGELOG.md` with release version and date, ensuring that all relevant changes are reflected.
+- [ ] Update the version in all places with `./update-version.sh <version-here>`
 - [ ] Freeze any new incremental sql scripts released in this version in `incremental_freeze_test`
```
Comment (sumerman) [inline RELEASING.md:16]: this seems to be done by the update version script

**Review by sumerman — APPROVED** [review]
LGTM, but I have a question: why not have it as a template that build.rs replaces?

**Comment by JamesGuthrie** [comment]
> why not have it as a template that build.rs replaces?

I'm not sure that I follow. Could you explain what you're thinking?

File: RELEASING.md (hunk 1 above)
Comment (sumerman) [reply RELEASING.md:16]: right

**Comment by sumerman** [comment]
> > why not have it as a template that build.rs replaces?
> 
> I'm not sure that I follow. Could you explain what you're thinking?

In most cases, we are altering the version string in the module name. For generated SQL, this is done by PGX, but since we are in charge of hand-written.sql, we should be able to substitute a set string with the expected module name.

**Comment by JamesGuthrie** [comment]
> > > why not have it as a template that build.rs replaces?
> > 
> > 
> > I'm not sure that I follow. Could you explain what you're thinking?
> 
> In most cases, we are altering the version string in the module name. For generated SQL, this is done by PGX, but since we are in charge of hand-written.sql, we should be able to substitute a set string with the expected module name.

Ah, I see what you mean. I guess your question is essentially "Why didn't you implement this as https://github.com/timescale/promscale_extension/issues/350"?. I don't have a good answer. I think that I thought "we haven't done it for so long, it must be complicated".

**Review by jgpruitt — APPROVED** [review]
Very cool.

**Pull request merged**